data/tlg0062/tlg001/__cts__.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0062" projid="greekLit:tlg001" urn="urn:cts:greekLit:tlg0062.tlg001" xml:lang="grc">
-            
+
    <ti:title xml:lang="eng">Phalaris</ti:title>
-   
+
    <ti:edition urn="urn:cts:greekLit:tlg0062.tlg001.perseus-grc2" workUrn="urn:cts:greekLit:tlg0062.tlg001" xml:lang="grc">
       <ti:label xml:lang="grc">Φάλαρις</ti:label>
       <ti:description xml:lang="eng">Lucian, Vol. 1. Harmon, A. M., editor. London: William Heinemann, Ltd.; Cambridge, MA: Harvard University Press, 1913.</ti:description>
    </ti:edition>
-   
-</ti:work>
\ No newline at end of file
+
+   <ti:translation urn="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0062.tlg001">
+      <ti:label xml:lang="eng">Phalaris</ti:label>
+      <ti:description xml:lang="eng">Lucian, Vol. 1. Harmon, A. M., editor. London: William Heinemann, Ltd.; Cambridge, MA: Harvard University Press, 1913.</ti:description>
```

Comment:
Harmon is the translator here, not editor

Reply:
I understand the importance of consistency but expanding the initials helps the reader and I found that information because I needed it. (It took a while to track down the K. in "K. Kilburn"). So I am going to keep that and any birth/death dates I find. Libraries are not consistent on this either.

Reply:
hey @gregorycrane I think it is that it ends up looking strange in the header and this is one reason for better integration with the catalog where all of this metadata can be found typically.